decidim-admin/app/views/decidim/admin/moderations/reports/index.html.erb

```diff
@@ -0,0 +1,105 @@
+      <dd><%= moderation.reportable.id %></dd>
+
+      <% if translatable_resource?(moderation.reportable) %>
+        <dt><%= t('.content_original_language') %></dt>
+        <dd><%= locale_name moderation.reportable.content_original_language %> </dd>
+      <% end %>
+
+      <dt>
+        <%= t('.reported_content') %>
+        <% if translatable_resource?(moderation.reportable) %>
+          <button
+            class="toggle-content"
+            data-see-original-button-label="<%= t('.see_original') %>"
+            data-see-current-button-label="<%= t('.see_current') %>"
+          >
+            <%= t('.see_original') %>
+          </button>
+        <% end %>
+      </dt>
+      <dd class="reported-content">
+        <div class="current">
+          <%= reported_content_for moderation.reportable %>
+        </div>
+        <% if translatable_resource?(moderation.reportable) %>
+          <div class="original">
+            <%= reported_content_for moderation.reportable, locale: moderation.reportable.content_original_language %>
+          </div>
+        <% end %>
+      </dd>
+
+      <dt><%= t('.author') %></dt>
+      <dd>
+        <%= reportable_author_name(moderation.reportable) %>
+      </dd>
+
+      <dt><%= t("models.moderation.fields.reportable_type", scope: "decidim.moderations") %></dt>
+      <dd><%= moderation.reportable.class.name.demodulize %></dd>
+
+      <dt><%= t("models.moderation.fields.created_at", scope: "decidim.moderations") %></dt>
+      <dd><%= l(moderation.created_at, format: :long) %></dd>
+
+      <dt><%= t('.participatory_space') %></dt>
+      <dd><%= translated_attribute moderation.participatory_space.title %></dd>
+      </div>
+    </dl>
+    <div class="table-scroll">
+      <table class="table-list">
+        <thead>
+          <tr>
+            <th><%= t("models.report.fields.reason", scope: "decidim.moderations") %></th>
+            <th><%= t("models.report.fields.locale", scope: "decidim.moderations") %></th>
+            <th><%= t(".reported_content") %></th>
+            <th><%= t("models.report.fields.details", scope: "decidim.moderations") %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% reports.each do |report| %>
+            <tr data-id="<%= report.id %>">
+              <td><%= t("decidim.admin.moderations.report.reasons.#{report.reason}") %></td>
+              <td><%= report.locale.present? ? locale_name(report.locale) : '' %></td>
+              <td>
+                <%= reported_content_excerpt_for(moderation.reportable) %>
+                <%= icon_link_to "fullscreen-enter",
+                                  moderation_report_path(moderation_id: moderation, id: report.id),
+                                  t("actions.expand", scope: "decidim.moderations"),
+                                  class: "action-icon--expand",
+                                  method: :get %>
```

Comment:
Please, remove this `method: :get` beause it'sc ausing a form submission and the back button is not working. Yay, Rails.